src/ast/value.rs

```diff
@@ -97,6 +97,37 @@ pub enum Value {
     Placeholder(String),
 }
 
+impl Value {
+    /// Get the string value of a `Value` wrapping a string.
+    /// This includes all quotation styles
+    /// `{Single,Double,TripleSingle,TripleDouble}{QuotedString,RawString,ByteString}`
+    /// as well as `EscapedStringLiteral`, `UnicodeStringLiteral`, `NationalStringLiteral`,
+    /// `HexStringLiteral`, `DollarQuotedString`.
+    /// This will panic if called on a non-string variant like `Value::Number`` or `Value::Null`.
+    pub fn as_str(self) -> String {
```

Comment:
```suggestion
    pub fn into_string(self) -> Option<String> {
```
The `into` prefix since this is consuming self and returning a new value (vs reference).
Then we want this to return None vs panic if the underlying literal isn't a string